microsoft-edge/toc.yml

```diff
@@ -1160,87 +1160,90 @@
+              href: ./progressive-web-apps-chromium/how-to/digital-goods-api.md
 
+        - name: Update a PWA and alert the user with badges, notifications, and push messages
+          items:
             - name: Synchronize and update a PWA in the background
-              href: progressive-web-apps-chromium/how-to/background-syncs.md
+              href: ./progressive-web-apps-chromium/how-to/background-syncs.md
 
-            - name: Handle files in Progressive Web Apps
-              href: progressive-web-apps-chromium/how-to/handle-files.md
+            - name: Re-engage users with badges and notifications
+              href: ./progressive-web-apps-chromium/how-to/notifications-badges.md
 
-            - name: Handle links to your Progressive Web App
-              href: progressive-web-apps-chromium/how-to/handle-urls.md
-              displayName: handle urls
+            - name: Re-engage users with push messages
+              href: ./progressive-web-apps-chromium/how-to/push.md
 
-            - name: Handle protocols in Progressive Web Apps
-              href: progressive-web-apps-chromium/how-to/handle-protocols.md
+        - name: Handle files, links, and protocols
+          items:
+            - name: Handle files in a Progressive Web App
+              href: ./progressive-web-apps-chromium/how-to/handle-files.md
 
-            - name: Display content in the title bar area using Window Controls Overlay
-              href: progressive-web-apps-chromium/how-to/window-controls-overlay.md
+            - name: Handle links to a Progressive Web App
+              href: ./progressive-web-apps-chromium/how-to/handle-urls.md
+              displayName: handle urls
 
-            - name: Share content with other apps
-              href: progressive-web-apps-chromium/how-to/share.md
+            - name: Handle protocols in a Progressive Web App
+              href: ./progressive-web-apps-chromium/how-to/handle-protocols.md
 
-            - name: Define app shortcuts
-              href: progressive-web-apps-chromium/how-to/shortcuts.md
+        - name: Define icons and a theme color
+          href: ./progressive-web-apps-chromium/how-to/icon-theme-color.md
 
-            - name: Build PWA-driven widgets
-              href: progressive-web-apps-chromium/how-to/widgets.md
+        - name: Store data on the device
+          href: ./progressive-web-apps-chromium/how-to/offline.md
 
-            - name: Build PWAs for the sidebar in Microsoft Edge
-              href: progressive-web-apps-chromium/how-to/sidebar.md
+        - name: Best practices for PWAs
+          href: ./progressive-web-apps-chromium/how-to/best-practices.md
 
-            - name: Provide in-app purchases with the Digital Goods API
-              href: progressive-web-apps-chromium/how-to/digital-goods-api.md
+        - name: Experimental features and origin trials for PWAs
+          href: ./progressive-web-apps-chromium/how-to/origin-trials.md
```

Comment:
I think there was a time when we had PWA origin trials listed in this article. We don't, anymore. And this page now acts mostly as a (possible useless) step before going to https://learn.microsoft.com/en-us/microsoft-edge/origin-trials/, where the real information is.

I suggest deleting this article.

Reply:
apprehensive about expanding scope of PR to include deleting .md files.  normally out of scope for a TOC redesign PR - usually create a PR to focus on a particular .md - adding or deleting or rewriting body content.

Reply:
result:
```
        - name: Experimental features and origin trials for PWAs  # todo: create PR to delete .md file
          href: ./progressive-web-apps-chromium/how-to/origin-trials.md
```